<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrokerManageClass</class>
 <widget class="QDialog" name="BrokerManageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>期货公司管理</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTableWidget" name="BrokerTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>460</width>
     <height>220</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>21</number>
   </attribute>
   <column>
    <property name="text">
     <string>经纪服务器名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>经纪商编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>交易服务期地址</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>经纪服务器名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>240</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>经纪商编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>交易服务期地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brokerName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="brokerID">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="frontAddr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>insertButton</sender>
   <signal>clicked()</signal>
   <receiver>BrokerManageClass</receiver>
   <slot>insertBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>BrokerManageClass</receiver>
   <slot>deleteBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>BrokerManageClass</receiver>
   <slot>updateBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrokerTable</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>BrokerManageClass</receiver>
   <slot>CellClicked(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BrokerManageClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insertBtnClicked()</slot>
  <slot>deleteBtnClicked()</slot>
  <slot>updateBtnClicked()</slot>
  <slot>CellClicked(int,int)</slot>
 </slots>
</ui>
